app/main/routes.py

```diff
@@ -24,7 +25,12 @@ def upload():
     local_authorities, place_names = check_authorised()
 
     if request.method == "GET":
-        return render_template("upload.html", local_authorities=local_authorities)
+        return render_template(
+            "upload.html",
+            local_authorities=local_authorities,
+            days_remaining=calculate_days_remaining(),
```

Comment:
similarly can change the param name here as well to `days_to_deadline` or something